src/lib/config.js

```diff
@@ -18,8 +18,10 @@ module.exports = {
       CHECK_DETAILS: "/confirm-details",
     },
     ANALYTICS: {
-      ID: process.env.GTM_ID,
+      GTM_ID_UA: process.env.UNIVERSAL_ANALYTICS_GTM_CONTAINER_ID,
```

Comment:
What do you think about putting some default values here so that it's set up for local development if the values don't exist?

Reply:
We have all the default values in env.sample, happy to add them here also if it is preferable

Reply:
Ah nice env.sample is good 👍